internal/scm/releaser.go

```diff
@@ -37,10 +37,15 @@ type StrategyConf struct {
 	Type string `yaml:"type,omitempty" validate:"oneof=pre-release full-release"`
 }
 
+type ChangelogConf struct {
+	Type string `yaml:"type,omitempty" validate:"oneof=legacy github"`
+}
+
 type Config struct {
-	TargetBranch string       `yaml:"targetBranch,omitempty" validate:"required"`
-	Labels       LabelsConfig `yaml:"labels,omitempty"`
-	Strategy     StrategyConf `yaml:"strategy,omitempty"`
+	TargetBranch string        `yaml:"targetBranch,omitempty" validate:"required"`
```

Comment:
omitempty and required?

Reply:
Yeah... that's a bit weird
```suggestion
	TargetBranch string        `yaml:"targetBranch" validate:"required"`
```